### ISSUE TITLE
aspect ratio not honored 

### ISSUE THREAD

**Issue by wdwatkins** [USGS-R/gsplot#367]
The `asp` argument is not being honored in plotting function calls.

``` r
set.seed(2576)
X <- runif(33)
Y<-seq(0,2,length.out = 33)
gs <- gsplot() %>% points(X, Y,
                          asp=1,frame.plot=FALSE) %>% axis(side=c(3,4),labels=FALSE) %>% grid()
gs
```

Produces this: 
![image](https://cloud.githubusercontent.com/assets/13987349/16500838/02d85b18-3ecc-11e6-854b-24001c772ab9.png)
`asp=1` should produce 1:1 axes.

**Comment by wdwatkins** [comment]
`labels` seems to be having the same problem.

**Comment by jordansread** [comment]
@wdwatkins when you make examples, can you make sure they are reproducible for us? I think that just means you need to include `X` and `Y` in your code snippet, but that helps us diagnose.

**Comment by jordansread** [comment]
not sure that `labels` is broken. You are saying "no labels" on sides 3 and 4, and there aren't labels on those sides.

**Comment by jordansread** [comment]
try triple ticks w/ r specified:
this
![image](https://cloud.githubusercontent.com/assets/2349007/16503237/d036ddfe-3ed7-11e6-8e80-8d3e2b98ac2b.png)
gives you this:

``` r
this.variable <- result(data=c(2,3,1),flags='three')
```

and that way you can use return lines too

**Comment by wdwatkins** [comment]
Sorry, X and Y are just arbitrary things to plot.  I added them.

To be more specific with labels: it works in the `axis()` call like above, but not when its inside `points()`, the same way `asp` doesn't.  Same X and Y as above.
`gs <- gsplot() %>% points(X, Y, labels=FALSE
                          ) %>% axis(side=c(3,4),labels=FALSE)`
![image](https://cloud.githubusercontent.com/assets/13987349/16503363/52fbb962-3ed8-11e6-9c50-264db72be1b8.png)
I'm trying to suppress initial labels on the main axes, so I can relabel them with an `axis()` call that places the labels between tick marks for the time/date axis labeling.

**Comment by lindsayplatt** [comment]
You shouldn't need to suppress them in order to change the labels:

``` r
gs <- gsplot() %>% 
    points(1:10,1:10) %>% 
    axis(side=1,at=c(2,4,6,8,10),labels=TRUE)
gs
```

![image](https://cloud.githubusercontent.com/assets/13220910/16503713/e373a242-3ed9-11e6-9b01-baca50b2ecc2.png)

**Comment by wdwatkins** [comment]
That also moves the tick marks though.  I'm trying to keep the original ticks, and only add labels.  To get something like this:

![image](https://cloud.githubusercontent.com/assets/13987349/16504145/0c15555e-3edc-11e6-8a00-b0465fa7847d.png)

I'm using this method: [http://stackoverflow.com/questions/7530933/placing-the-x-axis-labels-between-period-ticks](url).  Maybe there's a better option though.

**Comment by jordansread** [comment]
@wdwatkins do you want:

``` r
set.seed(2576)
X <- runif(33)
Y<-seq(0,2,length.out = 33)
gs <- gsplot() %>% points(X, Y, axes=FALSE ) %>% axis(side=c(1:4), labels=FALSE)
gs
```

That would be my suggestion for what you are doing...unfortunately, that doesn't actually work as expected for me:

![image](https://cloud.githubusercontent.com/assets/2349007/16526225/c8c3d3e4-3f75-11e6-8be8-6e86bd67f241.png)

This seems like a bug, right @lindsaycarr ?

**Comment by lindsayplatt** [comment]
Yes, that does seem like a bug. Something with axes=FALSE because this works as expected:

``` r
set.seed(2576)
X <- runif(33)
Y<-seq(0,2,length.out = 33)
gs <- gsplot() %>% points(X, Y) %>% axis(side=c(1:4), labels=FALSE)
gs
```

![image](https://cloud.githubusercontent.com/assets/13220910/16526467/0209b46a-3f77-11e6-9bf8-23165cdc2465.png)

**Comment by lindsayplatt** [comment]
See #370

**Comment by lindsayplatt** [comment]
`asp` still not working. Is this at all related to #366?

**Comment by lindsayplatt** [comment]
Saving for later because it's complicated (asp tells us to ignore one of the axis' limits) and not useful for repgen. Would be nice to have for ease of meeting SPN compliance.
